src/components/LoginPage/Slide2.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import imageMindmap from 'components/LoginPage/images/slide-2-mindmap.svg';
+import { Img, Caption } from 'components/LoginPage/slideComponents';
+
+const Slide = styled.section`
+  padding-bottom: 10%;
+  text-align: center;
+`;
+const H3 = styled.h3`
+  padding-bottom: 7%;
+  font-size: calc(13px + (24 - 13)*(100vw - 375px)/(1200 - 375));
```

Comment:
```
font-size: calc(13px + (24 - 13)*(100vw - 375px)/(1200 - 375));
```
is duplicated in `LoginPageComponent.js` and `Slide1.js`.

If we'd want to change it, it will most probably lead to changing it in one place and forgetting to change in another place.

Let's have all the font-sizes in variables and in one place.
E.g.
```
const fontSizeH3 = 'calc(13px + (24 - 13)*(100vw - 375px)/(1200 - 375));'
const fontSizeH2 = 'calc(16px + (30 - 16)*(100vw - 375px)/(1200 - 375));'
const fontSizeH1 = 'calc(24px + (45 - 24)*(100vw - 375px)/(1200 - 375));'
```
So we can easily see how many different font sizes do we have, how they differ, and easily updated them.

Reply:
I found that used `vmin` units is more appropriative to our task. Here some description 
[Specification](https://www.w3.org/TR/css-values-4/#vmin)
[Vmin on WebReference](https://webref.ru/css/unit/vmin)

Done in commit [db8d5d2](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/db8d5d213e208ca30de77f1c6674c40053f31f87)